### DISCUSSION TITLE
Toolset support status for cryogenic carbon capture

### DISCUSSION THREAD

**Discussion by sykde** [CCSI-Toolset/FOQUS#1250] [General]
Hello everyone,

Can the existing code be used to simulate cryogenic carbon capture (either through compression of flue gases or an external cooling loop)? If not, does anybody know what is missing in order to add this functionality?

**Comment by bpaul4** [comment]
Hello @sykde, at this time there is not a model supporting cryogenic carbon capture within the toolset.
